spark-operator/src/main/resources/EcsLayout.json

```diff
@@ -0,0 +1,49 @@
+{
+  "@timestamp": {
+    "$resolver": "timestamp",
+    "pattern": {
+      "format": "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'",
+      "timeZone": "UTC"
+    }
+  },
+  "ecs.version": "1.2.0",
```

Comment:
It's too ancient. Let's use the latest one in Apache Spark repo.
- https://github.com/elastic/ecs-logging-java/releases/tag/v1.6.0